tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -333,26 +333,20 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_ecc_evp_generate_ephemeral_key(&conn->secure.client_ecc_evp_params[0]));
 
             const uint8_t psk_data[] = "test";
-            struct s2n_psk *test_psks[S2N_TEST_PSK_COUNT] = { 0 };
             for (size_t i = 0; i < S2N_TEST_PSK_COUNT; i++) {
                 struct s2n_psk *psk = NULL;
                 EXPECT_OK(s2n_array_pushback(&conn->psk_params.psk_list, (void**) &psk));
                 EXPECT_SUCCESS(s2n_psk_init(psk, S2N_PSK_TYPE_EXTERNAL));
                 EXPECT_SUCCESS(s2n_psk_new_identity(psk, psk_data, sizeof(psk_data)));
                 EXPECT_NOT_EQUAL(psk->identity.size, 0);
                 EXPECT_NOT_EQUAL(psk->identity.data, NULL);
-                test_psks[i] = psk;
             }
             EXPECT_EQUAL(conn->psk_params.psk_list.len, S2N_TEST_PSK_COUNT);
 
             EXPECT_SUCCESS(s2n_tls13_handle_handshake_secrets(conn));
 
-            /* Verify all PSKs are wiped */
-            for (size_t i = 0; i < S2N_TEST_PSK_COUNT; i++) {
-                struct s2n_psk *psk = test_psks[i];
-                EXPECT_EQUAL(psk->identity.size, 0);
-                EXPECT_EQUAL(psk->identity.data, NULL);
-            }
+            /* Verify PSKs are wiped */
+            EXPECT_EQUAL(conn->psk_params.psk_list.mem.allocated, 0);
```

Comment:
Can you add EXPECT_NOT_EQUAL(conn->psk_params.psk_list.mem.allocated, 0) to a line above 346? That makes it a little easier to understand what is being tested.